### ISSUE TITLE
[BUG] adding --backed does not reduce memory consumption for s3 files

### ISSUE THREAD

**Issue by arogozhnikov** [chanzuckerberg/cellxgene#2487]
**Describe the bug**

Cellxgene processes fail when I try to open resonably-sized files (~3GB) from s3.

I've tried using --backed mode, but it did not reduce memory consumption.

**Tested**

1. no backed, s3 -> peak memory ~7 GB
2. backed, s3 -> peak memory ~7 GB
3. backed, copied file locally -> peak memory ~3.5 GB

(I believe even 3 GB is still high, but that's not the point of this issue).

In case it is important, data is in csc format (dtype is float32).

**Setup** 

Cellxgene==1.0.0

**Comment by arogozhnikov** [comment]
how peak memory consumption is measured: 
```
/usr/bin/time -v cellxgene launch ...
```

**Comment by MaximilianLombardo** [comment]
Hey @arogozhnikov thanks for the issue - could you also provide the stack trace to give some more details on which processes failed?

**Comment by arogozhnikov** [comment]
> could you also provide the stack trace to give some more details on which processes failed?

unfortunately, that happens in the cloud on ecs fargate (these instances also lack swap) and I don't have introspection there, so can't provide any traces

**Comment by arogozhnikov** [comment]
@MaximilianLombardo yes, so that's what I've done above - results are from local machine (mac os if that's important)

**Comment by MaximilianLombardo** [comment]
@arogozhnikov oh awesome - can you post the output in the terminal after you launched cellxgene here?
